storm-server/src/main/java/org/apache/storm/daemon/supervisor/Supervisor.java

```diff
@@ -528,6 +532,9 @@ void killWorkers(Collection<String> workerIds, ContainerLauncher launcher) throw
                 long start = Time.currentTimeMillis();
                 while (!k.areAllProcessesDead()) {
                     if ((Time.currentTimeMillis() - start) > 10_000) {
+                        if (killErrorMeter != null) {
+                            killErrorMeter.mark();
+                        }
```

Comment:
If this happens, the supervisor will exit. What's the value of this meter

Reply:
Line 545 in the modified version of this file catches the exception.  We just log an error.

Reply:
What's the modified version?

Reply:
Supervisor.java that I modified, or line 539 of Supervisor.java on master currently.

Reply:
Ah missed that. Thanks